tests/testthat/test-class-comment.R

```diff
@@ -181,10 +181,9 @@ test_that("print comment", {
 
   c2 <- create_comment(text = "this is another comment",
                        author = "Marco Polo")
-
-  exp <- "Author: Marco Polo\nText:\n Marco Polo:\nthis is another comment\n\nStyle:\n\n\n\n\nFont name: Aptos Narrow\nFont size: 11\nFont color: #000000\n\n"
   got <- capture_output(print(c2), print = TRUE)
-  expect_equal(exp, got)
+  exp <- "Author: Marco Polo\nText:\n Marco Polo:\nthis is another comment\n\nStyle:\n\n\n\n\nFont name: Aptos Narrow\nFont size: 11\nFont color: #000000\n\n"
+  expect_equal(got, exp)
```

Comment:
_\*Sigh\*_, I fear that there will be many changes of the type
`equal(exp, got)` -> `equal(got,exp)`
and if you would have asked me, I would have told you, that I like them the way they are. I know that they are incorrect in what testthat requires, but I’m a statistician and not a data scientist. I make a theory and test it with data and don’t test my data to make up a theory 😄 

After all they are just tests and nobody but me interacts with them or looks at them.